bin/pycbc_compress_bank

```diff
@@ -72,11 +72,17 @@ parser.add_argument("--tolerance", type=float, default=0.001,
                      "added to the compressed waveform until its "
                      "interpolation has a mismatch <= this value. Default is "
                      "0.001.")
-parser.add_argument("--interpolation", type=str, default="linear",
+parser.add_argument("--interpolation", type=str, 
+                default="linear",
                 help="The interpolation to use for decompressing the "
-                     "waveforms for checking tolerance. Options are 'linear', "
-                     "or any interpolation recognized by scipy's interp1d "
-                     "kind argument. Default is linear.")
+                     "waveforms for checking tolerance. Options are "
+                     "'linear_same_freq_points', which assumes that the "
+                     "frequency samples of the amplitude and phase "
+                     "interpolants are the same, or any interpolation "
+                     "recognized by scipy's interp1d kind argument, which "
+                     "allows the frequency interpolant samples to be "
+                     "different for the amplitude and phase. "
+                     "Default is linear_same_freq_points.")
```

Comment:
The help message doesn't match the code. The default is `linear` but the help message says `linear_same_freq_points`. In any event, this should just be `linear`. This argument's only purpose is to determine the algorithm used for interpolating points, not how those points are determined. The latter is determined by the `compression-algorithm` argument. It doesn't make sense to use different amplitude/phase points for `spa` or `mchirp` compression; likewise it doesn't make sense to use the same points for the compression you are working on. When you add your compression algorithm, you can add the switch to different points based on what `compression-algorithm` is set to.